Pacemaker/pacemaker_remote_poc/README.md

```diff
@@ -0,0 +1,36 @@
+### Conceptual Overview
+
+- A regular pacemaker cluster with communication using corosync can be extended up to 32 nodes. With the pacemaker_remote service, High Availability clusters can be extended to include additional nodes beyond this limit.
+- The pacemaker_remote service can be operated as a physical node (remote node) or as a virtual node (guest node). Unlike normal cluster nodes, both remote and guest nodes are managed by the cluster as resources.
+- From the resource management point of view, they behave as regular cluster nodes.
```

Comment:
Are resources hosted on remote node visible in `pcs status` output?

Reply:
Yes they are. Like below format:

WebServer (ocf::heartbeat:apache): Started remote2
rabbitmq (ocf::rabbitmq:rabbitmq-server): Started remote1